package.json

```diff
@@ -3,12 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.2",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
+    "react-circular-progressbar": "^2.0.4",
```

Comment:
- [ ] [OPTIONAL] When using `npm install` an error is produced, kindly fix your dependency so the app is installed correctly as for now I couldn't install the packages without using `--force`

![Screenshot_20220625_180656](https://user-images.githubusercontent.com/25227022/175783757-4a0c901e-0d9f-47c2-96b4-3925e3bb0e96.png)

Reply:
@Ismailco Please, how do i fix this